src/NzbDrone.Core.Test/ParserTests/QualityParserFixture.cs

```diff
@@ -381,7 +401,10 @@ public void should_parse_quality_from_name(string title)
         [TestCase("[CR] Sailor Moon - 004 [48CE2D0F].avi")]
         public void should_parse_quality_from_extension(string title)
         {
-            QualityParser.ParseQuality(title).QualityDetectionSource.Should().Be(QualityDetectionSource.Extension);
+            var result = QualityParser.ParseQuality(title);
+
+            result.SourceDetectionSource.Should().Be(QualityDetectionSource.Extension);
```

Comment:
Maybe a test with resolution + extension. Which should result in Source=Extension & Resolution=Name. combo of `should_parse_resolution_from_name` and `should_parse_quality_from_extension`.